duckduckgo/iOS
# Translations for update of Logins to Passwords

Task/Issue URL: https://app.asana.com/0/0/1206095676589855/f
Tech Design URL:
CC:

**Description**:
Translations of copy updated as part of the E2E project updating 'Logins' to 'Passwords'



**Steps to test this PR**:
Note - Italian has some additional translations compared to other languages as Emanuele manually improved the translations in Smartling & I've included those improvements in this PR
1. Launch the app in another supported language
2. Lightly run through some autofill steps (save a new credential, fill a saved credential, visit the password screen) and confirm copy is localised 



**Copy Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
